pc_ble_driver_py/ble_driver.py

```diff
@@ -43,9 +43,10 @@
 import sys
 import time
 import traceback
+import queue
 import logging
 from abc import abstractmethod
-from threading import Lock
+from threading import Thread, Lock
```

Comment:
Is Lock module import needed?

Reply:
Yes, since observer_lock and api_lock are still in use.